lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -679,6 +679,23 @@ function dosomething_reportback_preprocess_entity(&$variables) {
     $variables['theme_hook_suggestions'][] = 'reportback';
     $entity = $variables['elements']['#entity'];
   }
+
+  if (variable_get('rogue_collection', FALSE)) {
+    $rogue_base_url = variable_get('dosomething_rogue_url', 'https://rogue.dosomething.org/api');
+    $apiUrl = $rogue_base_url . '/v1/reportbacks';
```

Comment:
That will be appended to the URL when fetching each page [in the user's browser](https://github.com/DoSomething/phoenix/blob/caa4158e89f13dcb9f8aa79c3e413e0d0c790e70/lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js#L117-L124)!